myfitnesspal/client.py

```diff
@@ -44,6 +44,9 @@ class Client(MFPBase):
         "protein": (Mass, "g"),
         "sodium": (Mass, "mg"),
         "sugar": (Mass, "g"),
+        "fiber": (Mass, "g"),
+        "potass.": (Mass, "mg"),
+        "kilojoules": (Energy, "Calorie"),
```

Comment:
I _believe_ this is the wrong mapping here -- that'll equate a kJ with a kcal; I think you might get the correct results here if you  change `Calorie` to `kJ`, but I haven't tested that!

Reply:
Hey you're totally right - I just checked and it is indeed in kJ. I honestly thought they had some mistake on their end but it was me! Will fix this asap.